.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
i would do weekly. Daily is too often IMO

Reply:
Considering the (tiny) amount of actions to bump, I don't think it matters.